dnf.spec

```diff
@@ -161,17 +187,13 @@ BuildRequires:  python2-rpm >= %{rpm_version}
 Requires:       python2-rpm >= %{rpm_version}
 Recommends:     rpm-plugin-systemd-inhibit
 %endif
```

Comment:
on RHEL rpm-plugin-systemd-inhibit lives in optional repo. do not require it.

Reply:
fixed